metaworld/envs/mujoco/sawyer_xyz/base.py

```diff
@@ -96,6 +96,8 @@ def __init__(
         self.discrete_goals = []
         self.active_discrete_goal = None
 
+        self._state_goal = None  # OVERRIDE ME
```

Comment:
you can use an abc.abstractmethod on a property to ensure this

Reply:
`self._state_goal` gets set all over the place, so I think a full & proper change might be more work than it's worth right now. I'm sure we'll find a better way of handling it as we continue cleaning/refactoring the code